site/configure.xml

```diff
@@ -254,6 +257,75 @@ CONFIG_FILE=/etc/rabbitmq/testdir/bunnies</pre>
             </tr>
 
             <tr>
+              <td>RABBITMQ_CONFIG_FILE</td>
+              <td>
+                <ul>
+                  <li><b>Generic UNIX</b> - <span class="path"><span class="envvar">$RABBITMQ_HOME</span>/etc/rabbitmq/rabbitmq</span>
+                  </li>
+                  <li><b>Debian</b> - <span class="path">/etc/rabbitmq/rabbitmq</span></li>
+                  <li><b>RPM</b> - <span class="path">/etc/rabbitmq/rabbitmq</span></li>
+                  <li>
+                    <b>Mac OS X (Homebrew)</b> - <span class="path"><span
+                    class="envvar">${install_prefix}</span>/etc/rabbitmq/rabbitmq</span>,
+                    the Homebrew prefix is usually <code>/usr/local</code>
+                  </li>
+                  <li><b>Windows</b> - <span class="path"><span class="envvar">%APPDATA%</span>\RabbitMQ\rabbitmq</span></li>
+                </ul>
+              </td>
+              <td>
+                Main RabbitMQ config file path without the <code>.conf</code>
+                (or <code>.config</code>, for the classic format) extension.
```

Comment:
This "without the extension" thing is so confusing. I know RabbitMQ does this for a long time, but can't we document that we require the full filename but still support the missing suffix internally?

The fact that Erlang takes a filename without its `.config` extension is a technical detail we don't need to expose to end users.